packages/app/src/types/contracts/index.ts

```diff
@@ -2,9 +2,7 @@
 /* tslint:disable */
 /* eslint-disable */
 export type { ExchangeContractAbi } from './ExchangeContractAbi';
-export type { SwayswapContractAbi } from './SwayswapContractAbi';
```

Comment:
You can revert this file to master. I think it happen because of the issue with forc v0.12 we chat about